### PR TITLE
update README.md and create PV CRD folder

### PR DESCRIPTION
**Pull request by JimBugwadia**
update README.md and create PV CRD folder

### CODE REVIEW

**Comment by enj** [comment]
Warnings LGTM

Fixes #2

File: README.md [hunk 1]
```diff
@@ -11,6 +13,22 @@ You can reach the maintainers of this project at:
...
+
+Once you've been a reviewer, you can request to become an approver by filling a PR on our OWNERS file and another approver will need to approve you.
+
+## Code of conduct
 
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+
+## Projects
+
+* [Policy Violation CRD](policy-violation-crd/README.md)
```
Comment (hannibalhuang) [inline README.md:30]: please add robert's guardian project as well: [Policy Formal Verification](https://github.com/rficcaglia/guardian/README.md)

**Comment by JimBugwadia** [comment]
@hannibalhuang - added! The plan was to have Robert submit a PR to Owners so he can manage the related artifacts and updates.

File: README.md [hunk 2]
```diff
@@ -11,6 +13,23 @@ You can reach the maintainers of this project at:
...
 - [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-wg-policy)
 
-### Code of conduct
+## Join this repo
+
+File a request at https://github.com/kubernetes/org to be added to @kubernetes-sigs, using the Template.
+
+Once you've been a member, when you are ready to become a reviewer of other people's code, file a PR on our OWNERS file and an approver will need to approve you.
+
+Once you've been a reviewer, you can request to become an approver by filling a PR on our OWNERS file and another approver will need to approve you.
```
Comment (ericavonb) [inline README.md:22]:
```suggestion
Once you've been a reviewer, you can request to become an approver by filing a PR on our OWNERS file and another approver will need to approve you.
```

**Comment by ericavonb** [comment]
/hold cancel
will have to re-lgtm anyways after change pushed

**Pull request merged**